src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -670,7 +670,7 @@ job_status_type torque_driver_parse_status(const char *qstat_file,
     }
 
     /* Parse the qstat output, looking only for requested job_id */
-    std::string job_id_label("Job Id: ");
+    std::string job_id_label("Job Id:");
```

Comment:
This puzzles me a bit. The Torque will this still work on Azure correctly then?
https://github.com/equinor/ert/blob/33873a9832efd8c66771e40d8050215ab8906803/src/clib/lib/job_queue/torque_driver.cpp#L692

Reply:
Yes, proven through the tests.